docs/usermanual.md

```diff
@@ -172,21 +172,44 @@ MQTT devices commands are always push. For HTTP Devices commands to be push they
 command will be poll. When using the HTTP transport, the command handling have two flavours:
 
 -   **Push commands**: The request payload format will be the one described in the UL Protocol description. The device
-    will reply with a 200OK response containing the result of the command in the UL2.0 result format.
+    will reply with a 200OK response containing the result of the command in the UL2.0 result format. Example
+    of the HTTP request sent by IOTA in the case of push command:
+    
+```
+POST http://[DEVICE_IP]:[PORT]
+fiware-service: smart
+fiware-servicepath: /streetligths
+content-type: text/plain
+
+Robot1@turn|left
+```
 
 -   **Polling commands**: in this case, the Agent does not send any messages to the device, being the later responsible
     of retrieving them from the IoTAgent whenever the device is ready to get commands. In order to retrieve commands
     from the IoT Agent, the device will send the query parameter 'getCmd' with value '1' as part of a normal measure. As
     a result of this action, the IoTAgent, instead of returning an empty body (the typical response to a measurement
     report), will return a list of all the commands available for the device, sepparated by the character '#'. The
-    command payload is described in the protocol section (and its shared with the push commands). Whenever the device
+    command payload is described in the [commands syntax section](#commands-Syntax) (and its shared with the push commands). Whenever the device
     has completed the execution of the command, it will send the response in the same way measurements are reported, but
-    using the **command result format** as exposed in the [Protocol section](#protocol).
+    using the **command result format** as exposed in the [commands syntax section](#commands-Syntax).
 
 Some additional remarks regarding polling commands:
 
 -   Commands can be also retrieved without needed of sending a mesaure. In other words, the device is not forced to send
-    a measure in order to get the accumulated commands.
+    a measure in order to get the accumulated commands. Example to retrieve commands from IoT Agent:
+
+```text
+curl -X GET 'http://localhost:7896/iot/d?i=motion001&k=4jggokgpepnvsb2uv4s40d59ov&getCmd=1' -i
+```
+
+-   Example of the the HTTP response sent by IOTA in the case of polling commands:
+    
+```
+200 OK
+
+Robot1@turn|left#Robot1@move|20
```

Comment:
Is this correct? Or should be?

```
 Robot1@turn|left#move|20
```

I mean, `Robot1@` only one time.

Reply:
Not sure if several commands are allowed. No test about that were found in https://github.com/telefonicaid/iotagent-ul/blob/master/test/unit/ngsiv2/HTTP_commands_test.js

Reply:
Documentation above says:

> As a result of this action, the IoTAgent, instead of returning an empty body (the typical response to a measurement report), will return a list of all the commands available for the device, sepparated by the character '#'

My aim was to make an example aligned with documentation... Maybe we can change to the mono-command case if we have doubts, although probably the concept of pooling makes more sense when a queue of commands is waiting at IOTA.